docs/configuration.md

```diff
@@ -2670,6 +2670,9 @@ Spark uses [log4j](http://logging.apache.org/log4j/) for logging. You can config
 `log4j.properties` file in the `conf` directory. One way to start is to copy the existing
 `log4j.properties.template` located there.
 
+By default, Spark adds to the MDC 3 records: `appId`, `appName` and `taskName` you can add that to your patternLayout `%X{appId}` in order to print in the logs
```

Comment:
I think it's better to put the doc in `conf/log4j.properties.template`, where users use this feature.

Reply:
Maybe in both places?

Reply:
@cloud-fan What do you think?